### PR TITLE
Remove types_by_module view

### PR DESCRIPTION
**Pull request by esoergel**
Wait until https://github.com/dimagi/commcare-hq/pull/9784 is merged.  This PR only removes the view.

### CODE REVIEW

File: corehq/pillows/mappings/app_mapping.py [hunk 1]
```diff
@@ -1,8 +1,7 @@
 from corehq.util.elastic import es_index
 
-APP_INDEX = es_index("hqapps_12f505392bcec1a43d1c2c9ef9e0cb51")
-APP_MAPPING={'_meta': {'comment': 'Autogenerated [corehq.apps.app_manager.models.Application] mapping from ptop_generate_mapping 11/07/2013',
-           'created': None},
+APP_INDEX = es_index("hqapps_2016-01-05_1821")
+APP_MAPPING={'_meta': {'created': None},
```
Comment (dimagimon) [inline corehq/pillows/mappings/app_mapping.py]: E225 missing whitespace around operator

**Comment by dimagimon** [comment]
Could not review pull request. It may be too large, or contain no reviewable changes.

**Comment by esoergel** [comment]
Rebased off latest master so the PR only shows the actual changes.  This is g2m whenever a reindex is convenient.

**Comment by dimagimon** [comment]
Could not review pull request. It may be too large, or contain no reviewable changes.

**Comment by czue** [comment]
:skull: :airplane:

**Pull request merged**